Generals/Code/Libraries/Source/WWVegas/WW3D2/textureloader.cpp

```diff
@@ -1046,7 +1046,7 @@ void TextureLoadTaskClass::Begin_Texture_Load()
 			DDSFileClass dds_file(Texture->Get_Full_Path(),Get_Reduction());
 			if (dds_file.Is_Available()) {
 				// Destination size will be the next power of two square from the larger width and height...
-				unsigned width, height, depth;
+				unsigned width=0, height=0, depth=0; // Depth is not used.
```

Comment:
Maybe needs to be initialized with 1? That is how Zero Hour does it in `W3DShroud::init`.